app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -365,4 +370,10 @@ open class FenixApplication : LocaleAwareApplication() {
             Logger.error("Failed to initialize web extension support", e)
         }
     }
+
+    protected fun recordOnInit() {
+        // This gets called by more than one process. Ideally we'd only run this in the main process
+        // but the code to check which process we're in crashes because the Context isn't valid yet.
+        StartupTimeline.onApplicationInit()
```

Comment:
nit: The comment in `init` is warning that nothing should happen before this. However `recordOnInit()` looks fairly harmless and I wonder if it could happen that someone adds something in front of it inside this method. Is there any reason we can't inline the call to `StartupTimeline`?

Reply:
Good point. I wanted to:
1. avoid duplicating the comment, "this gets called by more than one process..."
1. (I care about this less) make it easy for additional `Application` classes to call this method, if necessary

@pocmo Is `MigratingFenixApplication` going away soon (merged into `FenixApplication`)? If so, I'll inline with duplicated comment. If not, I'll add a comment to this method not to add anything above that line.